### ISSUE TITLE
Loss function not converge and the prime predict is  not precise

### ISSUE THREAD

**Issue by AndreaGastaldi** [AndreaGastaldi/Tensorflow-Neural-Network#1]
HI, I build a neural network with regrission for predict an insurances prime. I have got a problem with a loss function because if I launch the model with a DB_1 ( that have 9 attribute) the loss converge correctly around the 0; if I launch the same model with DB_2 ( that have 40 attribute) the  loss function not coverge ( it stop around the value 1000). I can try to change learning rate parameter, number of hidden layer, number of batch and I used a different inizialized technique for weight and bias, but the result not change. I think that the causes of issue is in the ReLU activation function. There are another activation function for NNT with regression? Because Softmax and sigmoid are used in classification model.
